src/components/header.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import { Logo, SearchForm } from './common-small';
+import '../sass/components/_header.scss';
+
+function SearchHeader({ classes = '' }) {
+  return (
+    <header className={`header ${classes}`}>
+      <div className='header-inner'>
+        <Logo />
+        <SearchForm
+          formClass='search-form-one'
+          inputType='text'
+          inputId='product-name'
+          inputName='product-name'
+          inputClass='search-form-one__input'

```

Comment:
```suggestion
          type='text'
          id='product-name'
          name='product-name'
          class='search-form-one__input'
```
như này là được rồi, vì chắc chắn Search thì phải là input rồi
Giống như kiểu
```
const person = {
   personName: '',
   personAge: '',
}
```
Việc này là k cần thiết